### ISSUE TITLE
Upper and Lowers Bounds for Hierarchical Forecast Engine?

### ISSUE THREAD

**Issue by mgiangreco** [antoinecarme/pyaf#66]
Is it intentional that the hierarchical forecast engine does not produce upper and lower bound estimates, whereas the standard forecast engine does?

**Comment by antoinecarme** [comment]
Yes. Laziness/Priorities sometimes explain a lot of things. Planned.

**Comment by antoinecarme** [comment]
I am sure you will find where to start ;)

I usually start by prototyping all computation details in a jupyter notebook. create some python test scripts, document all, double check and then start real coding. I can help if you share your findings in each step.

Good luck.

**Comment by antoinecarme** [comment]
Not sure the development process can match your project requirements though. sorry.
